concent_api/gatekeeper/utils.py

```diff
@@ -10,6 +13,13 @@ def gatekeeper_access_denied_response(message, error_code = None, path = None, s
         'client_key':      client_key,
     }
 
+    logging.log_gatekeeper_access_denied_response_messages(method_origin,
```

Comment:
First argument shouldn't be in the same line as function call

Reply:
Plus please fix indentation below.